ext/Pod-Html/t/pod2html-lib.pl

```diff
@@ -41,20 +45,40 @@ sub convert_n_test {
     my $infile   = catpath $vol, $new_dir, "$podfile.pod";
     my $outfile  = catpath $vol, $new_dir, "$podfile.html";
 
-    my %default_args_table = (
+    my %args_table = (
         infile      =>    $infile,
         outfile     =>    $outfile,
         podpath     =>    't',
         htmlroot    =>    '/',
         podroot     =>    $cwd,
     );
-    my @default_args_list = map { "--${_}=" . $default_args_table{$_} }
-        keys %default_args_table;
-    # To add/modify args to p2h, use @p2h_args
-    Pod::Html::pod2html(
-        @default_args_list,
-        @p2h_args,
-    );
+    my %no_arg_switches = map { substr($_,2) => 1 } ( qw|
+           --flush --recurse --norecurse
+           --quiet --noquiet --verbose --noverbose
+           --index --noindex --backlink --nobacklink
+           --header --noheader --poderrors --nopoderrors
+    | );
+    if (defined $p2h_args_ref) {
+        for my $sw (keys %{$p2h_args_ref}) {
+            if ($no_arg_switches{$sw}) {
+                $args_table{$sw} = undef;
+            }
+            else {
```

Comment:
While the code is not extremely consistent, in general the pod2html code appears to cuddle elses, and I think we should _try_ to avoid introducing inconsistency.

Reply:
Addressed in commit 37ad10f992.